src/Service/Models/SignInTokenRequest.cs

```diff
@@ -0,0 +1,6 @@
+namespace Passwordless.Service.Models;
+
+public class SignInTokenRequest(string userId) : RequestBase
```

Comment:
Should this be a record maybe?

Reply:
I added `RequestBase` which is a class.  `RequestBase` has `Origin` and `RpId` which is on the other request endpoints.  Since RequestBase is a class, the request object has to be a class. That was the reason I switched it over.

Reply:
Can `RequestBase` be a record then too? 🤔 

This is me nitpicking though

Reply:
No worries about the nits. Glad we're talking it through. If we change the other request objects to records too, we can change request base to a record. Let's hold off though until this is merged in and then make that a separate pr.